### ISSUE TITLE
Add a "Getting started" page

### ISSUE THREAD

**Issue by erlend-sh** [doppioslash/arewegameyet#99]
I suggest adding a new top-level page titled "Getting started". Its purpose would be to provide a quick and safe answer to the many budding Rust game developers who are wondering "where should I start?"

@kvark already made one such write-up:

https://gist.github.com/kvark/840c8cadf755b0d822b331222b0c3095

I would prefer something a tad more fleshed out, but the essence of it is spot-on. It's also encouraging to see that a year later it is still good advice, which speaks to the growing stability of the ecosystem.

How to write the article as democratically as possible is of course an issue. A more elaborate process could be considered for the future, but for now I would suggest the following:

- Start a new issue: "RFC: Getting started 09-2018", provide a skeleton similar to kvark's and solicit feedback from the Rust community. Invite engine creators and game developers to make their case.
- Once the discussion has gone on for two weeks or so (which might not be much discussion at all at this early stage) the AWGY team makes an executive decision and publishes the final draft to the website.
- The "Getting started" page will link back to the draft discussion for transparency.

**Comment by doppioslash** [comment]
@erlend-sh  Agreed on adding a Getting Started article.
Though isn't that whole procedure a tad overcomplicated for one simple article?
I'd rather just draft one, and then people can send pull requests to tweak it, or entirely change it or whatever.

**Comment by erlend-sh** [comment]
Yeah I agree, I was getting a bit ahead of myself. Best to just start it off as a living document like you're suggesting.
